code/modules/computer3/computers/law.dm

```diff
@@ -24,7 +24,7 @@
 
 
 	attackby(obj/item/weapon/aiModule/module as obj, mob/user as mob, params)
-		if(!is_station_level(user.z))
+		if(is_away_level(user.z))
```

Comment:
You're going to have to check for if the person is on the same z-level as the `current`'s turf z-level.